crates/handlers/src/graphql/mutations/user.rs

```diff
@@ -323,6 +328,61 @@ impl SetPasswordPayload {
     }
 }
 
+/// The input for the `resendRecoveryEmail` mutation.
+#[derive(InputObject)]
+pub struct ResendRecoveryEmailInput {
+    /// The recovery ticket to use.
+    ticket: String,
+}
```

Comment:
I don't understand the intended flow for this.

I assumed the recovery ticket is the code that arrives in the e-mail (embedded in a URL, probably) and that lets you recover the account.

So why would you use a recovery ticket to resend the recovery e-mail?

If the ticket is more public than that, I don't think I am happy with the idea that you can use a ticket to request the username of the user...

edit: from the client code I think I gleam that it's for expired tickets. If so, can you state that more clearly in the documentation for the resend operation?

Reply:
That's right, I've added a comment in 8a69c9aa93b54894b42777be57d215ef03eb1c51